runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -1442,6 +1442,43 @@ TR_J9SharedCacheServerVM::isClassLibraryMethod(TR_OpaqueMethodBlock *method, boo
    return false;
    }
 
+TR_OpaqueClassBlock *
+TR_J9SharedCacheServerVM::getClassFromSignature(const char * sig, int32_t sigLength, TR_ResolvedMethod * method, bool isVettedForAOT)
+   {
+   return getClassFromSignature(sig, sigLength, (TR_OpaqueMethodBlock *)method->getPersistentIdentifier(), isVettedForAOT);
```

Comment:
Here `method` is a `TR_ResolvedMethod` and applying `getPersistentIdentifier()` to it may do the wrong think (not 100% sure). Maybe we should cast it to `TR_ResolvedJ9JITaaSServerMethod`